### DISCUSSION TITLE
Give Me Some Recipes!

### DISCUSSION THREAD

**Discussion by coltmay** [nss-day-cohort-47/C47-Discussions#33] [Topix.random]
Anybody else like to cook?  I'm making Gazpacho this week.  Simple, but delicious!
***
## Ingredients
- [ ] 4 cups tomato juice
- [ ] 1 onion, minced
- [ ] 1 green bell pepper, minced
- [ ] 1 cucumber, chopped
- [ ] 2 cups chopped tomatoes
- [ ] 2 green onions, chopped
- [ ] 1 clove garlic, minced
- [ ] 3 tablespoons fresh lemon juice
- [ ] 2 tablespoons red wine vinegar
- [ ] 1 teaspoon dried tarragon
- [ ] 1 teaspoon dried basil
- [ ] ¼ cup chopped fresh parsley
- [ ] 1 teaspoon white sugar
- [ ] salt and pepper to taste

## Directions
In a blender or food processor, combine tomato juice, onion, bell pepper, cucumber, tomatoes, green onions, garlic, lemon juice, red wine vinegar, tarragon, basil, parsley, sugar, salt, and pepper. Blend until well-combined but still slightly chunky. Chill at least 2 hours before serving.

**Comment by Chadwell-Clark** [comment]
### Though not a recipe,
 I have discovered Aleppo Pepper as a spice that find itself being added to many recipes. It is a like red pepper but is not as spicy and imparts a nice flavor. Combined with smoked paprika it can really be a great addition to chicken, fish, beef, pork and vegetables.

**Comment by SidneyCrandall** [comment]
## So not a super crazy recipe
### Chipotle Dressing 
- Ingredients
6 tablespoons of Lemon Juice
2 tablespoons Honey
1/2 cup Olive Oil 
1 or more Chipotle Peppers
1/2-1 tablespoon Adobo Sauce 
2 Garlic Cloves
1/2 teaspoon salt
1/2 teaspoon pepper
smoked paprika for taste

1) Combine all ingredients into a food processor or blender. Blend until smooth. Season to taste.
2) Can be used immediately or transferred to airtight container and kept for 4 days to a week.

**Reply by coltmay** [reply]
The thread has been revived!  Thank you, Sidney!  This sound delicious.  I think I may drizzle it over some roasted vegetables this weekend.  :)

**Comment by TarynL** [comment]
I have recently gone vegan and was craving fettuccine alfredo. This little take on it was quite a crowd pleaser. Top it with some crushed red pepper for a little kick!

**Creamy Vegan Mushroom Artichoke Paste**

- [ ] 8oz dry fettuccine pasta
- [ ] 2tsp salt
- [ ] 8oz sliced portobello mushrooms
- [ ] 1/2 sliced red onion
- [ ] 2 tsp olive oil
- [ ] salt and pepper to taste
ArtichokeSauce
- [ ] 1 jar of artichoke hearts(10-12oz)
- [ ] 1/2c  water
- [ ] 1/4c olive oil
- [ ] 10 fresh sage leaves
- [ ] 3 garlic cloves
- [ ] 1/2tsp salt and pepper
- [ ] lemon to taste

1. Bring big pot of generously salted water to a boil(for the pasta). Cook the pasta to al dente.
2. While water is boiling, in a large skillet, sauté the onions and mushrooms over medium heat in a little olive oil. Season with salt and pepper, set aside.
3. Make Artichoke Sauce-combine all ingredients in a food processor or blender. If using marinated artichokes, lemon may not be needed.
4. Add sauce to onions and mushrooms to heat up, then toss with pasta.
5. Serve with a salad, garlic bread or a la carte!!! ENJOY
